doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -41,6 +41,14 @@ Protocols
 This section provides detailed lists of changes by :ref:`protocol <protocols>`.
 See `Samples`_ for lists of changes for the protocol-related samples.
 
+Bluetooth LE
+------------
+
+* Added support for changing the radio transmitter's default power level using :c:func:`sdc_default_tx_power_set`.
+* Added support for changing peripheral latency mode using :c:func:`sdc_hci_cmd_vs_peripheral_latency_mode_set`.
+
+For details, see `softdevice_controller_changelog`.
```

Comment:
Please fix the link:
```rst
For details, see :ref:`nrfxlib:softdevice_controller_changelog`.
```
I got this syntax from the previous release notes (nrf/doc/nrf/releases/release-notes-1.0.0.rst).